src/writeData/components/AddPluginToConfiguration.tsx

```diff
@@ -30,84 +41,122 @@ interface AddPluginToConfigurationCTAProps {
+
+  return (
+    <>
+      <div className="write-data--details">
+        <Grid>
+          <Grid.Row>
+            <Grid.Column
+              widthXS={Columns.Twelve}
+              widthMD={Columns.Six}
+              widthLG={Columns.Three}
+            >
+              <div className="write-data--details-thumbnail">{thumbnail}</div>
+              <Dropdown
+                button={(active, onClick) => (
+                  <Dropdown.Button
+                    active={active}
+                    color={ComponentColor.Primary}
+                    style={{width: '100%'}}
+                    onClick={onClick}
                   >
-                    <span>Add to an existing configuration</span>
-                  </Dropdown.Item>
-                </Dropdown.Menu>
-              )}
-              style={{width: 'auto'}}
-              testID="add-plugin-to-configuration--dropdown"
-            />
-          </Grid.Column>
-          <Grid.Column
-            widthXS={Columns.Twelve}
-            widthMD={Columns.Six}
-            widthLG={Columns.Nine}
-          >
-            <div
-              className="write-data--details-content markdown-format"
-              data-testid="load-data-details-content"
+                    Use this plugin
+                  </Dropdown.Button>
+                )}
+                className="use-plugin--telegraf-configuration"
+                menu={onCollapse => (
+                  <Dropdown.Menu onCollapse={onCollapse}>
+                    <Dropdown.Item
+                      key="Create-new-configuration-telegraf-plugin"
+                      value="Create a new configuration"
+                      onClick={() => {
+                        history.push(
+                          `/${ORGS}/${orgID}/load-data/${TELEGRAF_PLUGINS}/${contentID}/new`
+                        )
+                      }}
+                      selected={false}
+                      testID="create-new-configuration-from-plugin--dropdown-item"
+                    >
+                      <span>Create a new configuration</span>
+                    </Dropdown.Item>
+                    {telegrafs?.length ? (
+                      <Dropdown.Item
+                        key="Add-to-existing-configuration-telegraf-plugin"
+                        value="Add to an existing configuration"
+                        onClick={() => {
+                          history.push(
+                            `/${ORGS}/${orgID}/load-data/${TELEGRAF_PLUGINS}/${contentID}/add`
+                          )
+                        }}
+                        selected={false}
+                        testID="add-to-existing-configuration-from-plugin--dropdown-item"
+                      >
+                        <span>Add to an existing configuration</span>
+                      </Dropdown.Item>
+                    ) : null}
+                  </Dropdown.Menu>
+                )}
+                style={{width: 'auto'}}
```

Comment:
another place to replace the object being recreated with a static object

Reply:
Updated.